cmd/baton-jira-datacenter/config.go

```diff
@@ -7,10 +7,16 @@ import (
 var (
 	instanceURLField = field.StringField("instance-url", field.WithRequired(true), field.WithDescription(`The URL that Jira is hosted at. Example: "https://localhost:8080"`))
 	accessTokenField = field.StringField("access-token", field.WithRequired(true), field.WithDescription("The Jira personal access token to authenticate with."))
+	projectKeysField = field.StringSliceField("project-keys", field.WithDescription("The Jira project keys to filter for external ticket provisioning."))
```

Comment:
The phrase `to filter for` feels ambiguous. 
- If you mean to exclude the keys that were passed in, I'd use `to filter out`
- Otherwise, borrowed from `baton-github`: `Limit external ticket provisioning to specific Jira projects by project key`